lib/SearchProvider.js

```diff
@@ -52,7 +52,15 @@ NewTabSearchProvider.prototype = {
 
   get state() {
     return Task.spawn(function*() {
-      let state = yield ContentSearch._currentStateObj();
+      // a new version of ContentSearch.jsm landed in nightly but not firefox
```

Comment:
This comment is outdated the moment it lands because it does not contain the specific firefox version number.
Please include the version.